templates/child_comments.twig

```diff
@@ -1,11 +1,11 @@
 <article class="single-comment {{ comment.comment_type }} comment-level-{{ comment.depth }}" id="comment-{{ comment.ID }}" >
-	<p itemprop="commentText" class="single-comment-text">{{ comment.comment_content|wpautop|striptags( '<br>,<p>' )|raw }}</p>
```

Comment:
The `itemprop` attribute didn't seem to be used for anything 🤷‍♀️ and the `|wpautop|striptags( '<br>,<p>' )` was causing issues in the markup